src/main/java/org/shanerx/tradeshop/util/Utils.java

```diff
@@ -118,60 +116,6 @@ public String getPrefix() {
         return PREFIX;
     }
 
-    /**
-     * Returns the Help permission.
-     *
-     * @return help
-     */
-    public Permission getHelpPerm() {
-        return PHELP;
-    }
-
-    /**
-     * Returns the Who permission.
-     *
-     * @return who
-     */
-    public Permission getWhoPerm() {
-        return PWHO;
-    }
-
-    /**
-     * Returns the normal {@code [Trade]} sign create permission.
-     *
-     * @return the Trade create permission
-     */
-    public Permission getCreatePerm() {
-        return PCREATE;
-    }
-
-    /**
-     * Returns the {@code [iTrade]} sign create permission.
-     *
-     * @return the iTrade create permission
-     */
-    public Permission getCreateIPerm() {
-        return PCREATEI;
-    }
-
-    /**
-     * Returns the {@code [BiTrade]} sign create permission.
-     *
-     * @return the BiTrade create permission
-     */
-    public Permission getCreateBiPerm() {
-        return PCREATEBI;
-    }
-
-    /**
-     * Returns the TradeShop admin destroy permission.
-     *
-     * @return the Trade create permission
-     */
-    public Permission getAdminPerm() {
-        return PADMIN;
-    }
-
```

Comment:
Uh... why dozens of lines of code being obliterated?
I assume because it is part of the Permissions enum... (?)

Reply:
correct